scripts/generate-eth2-validator-keys.sh

```diff
@@ -0,0 +1,12 @@
+cd ../tmp
+echo $PWD
+
+rm -rf ../testdata/layer2/pos/validator-keys
+
+tools/bin/eth2-val-tools keystores \
```

Comment:
I notice we already uploaded lots of keystore keys.  Can we use this script to generate the key on demand?